### ISSUE TITLE
Dismiss on the watch does not work

### ISSUE THREAD

**Issue by MrFly** [urbandroid-team/Sleep-as-Android-Garmin-Addon#20]
A tap on dismiss does not work. It simply does nothing (at least as user I'm not able to see any difference than not tapping on it).

I'm using SaA beta, and all other pieces of software in the current versions (10.12.2018) on a Garmin VA3.

This was [mentioned here](https://github.com/urbandroid-team/Sleep-as-Android-Garmin-Addon/issues/10#issuecomment-445772631)

**Comment by CWempe** [comment]
Same for me (vivoactive 3).

`Dismiss` and `Snooze` do not work.
The watch keeps vibration because of the alarm.
Only workaround: restart watch.

Here is a recording of the problem:

![garmin_alarm](https://user-images.githubusercontent.com/1270359/103557730-edc2a100-4eb3-11eb-8dc7-04cc8707d845.gif)

Are we the only ones with this issue?

**Comment by Artaud** [comment]
Thank you for reporting this - the idea is that dismiss sends a dismiss command to the phone, which in turn sends the dismiss command back to the watch, and only after that the alarm is dismissed. 
This is because you may have set up Captcha task on the phone, which needs to be solved prior to ending the alarm.

Also, if the connection to the phone is broken (phone turned off maybe), then the dismiss button would not dismiss the alarm as well.

Is any of these the case?

**Comment by CWempe** [comment]
I got this issue again today.

Just sent a wearable report.

* First I tried to dismiss via phone.
* 
* After ~20 seconds I rebooted the watch 
* I snoozed on the phone (10min)
* After 10min I closed the alarm on the phone by rating my sleep.
  * I forgot to explicitly "close" the alarm on the phone. I usually do that.
* A few seconds later the phone app (or the Garmin plugin?) still tried to show a notification about sleep tracking.
* It too a few attempts to "stop" this.

I am not 100% sure this steps are correct. I was not fully awake. 😄 

I do not have captchas enabled. Just "smart wake". And I use snooze a lot.


btw: A few days ago I saw the option to "force stopp app" which was a nice work around so I did not needed to restart the watch. Why is this gone already?
